layouts/partials/breadcrumbs.html

```diff
@@ -0,0 +1,26 @@
+{{ $url := replace .Permalink ( printf "%s" .Site.BaseURL) "" }}
```

Comment:
```suggestion
{{ $url := replace .Permalink .Site.BaseURL "" }}
```

Perhaps this can be simplified a bit